content/geoip/release-notes.mdx

```diff
@@ -7,6 +7,67 @@ title: GeoIP2 Release Notes
 Subscribe to the [GeoIP2 release notes RSS feed](/geoip/release-notes/rss.xml).
 </Alert>
 
+<ReleaseNote date="2022-01-25" title="Mobile country and mobile network code data">
+We have added data containing the mobile country code and mobile network code to
+several of our GeoIP2 product and services. Mobile country code and mobile
+network code data uniquely identify mobile network operators (carriers). This
+data can be used for carrier segmentation and targeting as well as general
+analytics. [Learn more about mobile country code data on
+Wikipedia.](https://en.wikipedia.org/wiki/Mobile_country_code)
+
+This data will be included in the following GeoIP2 products and services:
+
+ - GeoIP2 ISP database
+ - GeoIP2 Enterprise database
+ - GeoIP2 City Plus web service
+ - GeoIP2 Insights web service
+ - minFraud Insights web service
+ - minFraud Factors web service
+
+### Database users
+The `mobile_country_code` and `mobile_network_code` data will be included in all
+new releases of the GeoIP2 ISP and Enterprise databases starting with today's
+release.
+
+#### CSV file users
+The new data columns will be added to the following files:
+
+- GeoIP2 ISP database
+  - `GeoIP2-ISP-Blocks-IPv4.csv`
+  - `GeoIP2-ISP-Blocks-IPv6.csv`
+- GeoIP2 Enterprise database
+  - `GeoIP2-Enterprise-ISP.csv`
+
+See our developer documentation for the GeoIP2
+[ISP](/geoip/docs/databases/isp?lang=en#blocks-files) and
+[Enterprise](/geoip/docs/databases/enterprise?lang=en#isp-file) databases for
+more information about the files. You can also download a sample CSV database
+containing the new data columns for GeoIP2
+[ISP](/geoip/docs/databases/isp?lang=en#example-files) and
+[Enterprise](/geoip/docs/databases/enterprise?lang=en#example-files) from our
+developer's site.
```

Comment:
```suggestion
developer portal.
```

Reply:
i've been moving to "developer's site." there's an open discussion about what the preferred way to refer to the site should be in another document, and if you have a strong feeling one way or another i would appreciate your input. is it okay to proceed with this language for now, and then we'll standardize after the discussion is resolved?

Reply:
Sure, we can merge this. What got my attention was the use of the possessive here, since it sounds like it a site for a single developer or a site that belongs to a developer rather than just a resource for developers. So, I just checked the front page of the dev site to see what we call it there. :)